### PR TITLE
add Denso case study and fix svg images at top of case study landing page

### PR DESCRIPTION
**Pull request by alexcontini**

### CODE REVIEW

File: content/en/case-studies/appdirect/index.html [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 css: /css/style_case_studies.css
-logo: appdirect_featured_logo.png
+logo: appdirect_featured_logo.svg
 featured: true
-weight: 4
-quote: >
-   We made the right decisions at the right time. Kubernetes and the cloud native technologies are now seen as the de facto ecosystem.
 ---
 
 <div class="banner1" style="background-image: url('/images/CaseStudy_appdirect_banner1.jpg')">
```
Comment (sftim) [inline content/en/case-studies/appdirect/index.html:11]: @alexcontini would it be OK to separate this fix into its own pull request? The PR title implies that this PR only touches the Denso case study.

**Comment by alexcontini** [comment]
@sftim  sounds good - I just put that back to how it was. I did update the images so they aren't blurry at the top of the case study page in this PR as well, so I updated the title of this PR to include that

**Comment by zacharysarah** [comment]
@alexcontini 👋 LGTM! Can you please squash your commits so we can merge?
